src/index.ts

```diff
@@ -553,11 +553,13 @@ export class UnleashClient extends TinyEmitter {
 
     private async fetchToggles() {
         if (this.fetch) {
+            const oldController = this.abortController;
+            this.abortController = this.createAbortController?.();
```

Comment:
Always create a fresh instance first, then abort the old one, then use the new signal.